pkg/cmd/build/build.go

```diff
@@ -71,5 +73,13 @@ func Run(ctx context.Context, namespace, buildKitHost string, isOktetoCluster bo
 		analytics.TrackBuildTransientError(buildKitHost, success)
 		return err
 	}
+	if err != nil {
+		if registry.IsLoggedIntoRegistryButDontHavePermissions(err) || registry.IsNotLoggedIntoRegistry(err) {
+			imageTag, imageRegistry := registry.SplitRegistryAndImage(tag)
+			err = okErrors.UserError{E: fmt.Errorf("You are not authorized to push image '%s'.", imageTag),
+				Hint: fmt.Sprintf("Login into '%s' and verify that you have permissions to push images.", imageRegistry)}
```

Comment:
@rberrelleza I would also mention that they can use a different image where they have push permissions.
How would you rephrase it?